arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json

```diff
@@ -1290,59 +1425,65 @@
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the last access time as ticks since the epoch."
+          "description": "the last access time as ticks since the epoch."
         },
         "blockSize": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the block size for the file."
+          "description": "the block size for the file."
         },
         "childrenNum": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the number of children in the directory."
+          "description": "the number of children in the directory."
+        },
+        "expirationTime": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int64",
+          "description": "Gets the expiration time, if any, as ticks since the epoch. If this is set to 0 or DateTime.MaxValue there is no expiration."
         },
         "group": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the group owner."
+          "description": "the group owner."
         },
         "length": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the number of bytes in a file."
+          "description": "the number of bytes in a file."
         },
         "modificationTime": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the modification time as ticks since the epoch."
+          "description": "the modification time as ticks since the epoch."
         },
         "owner": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the user who is the owner."
+          "description": "the user who is the owner."
         },
         "pathSuffix": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the path suffix."
+          "description": "the path suffix."
         },
         "permission": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the permission represented as an string."
+          "description": "the permission represented as an string."
         },
         "type": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the type of the path object.",
+          "description": "the type of the path object.",
           "enum": [
-            "File",
-            "Directory"
+            "FILE",
+            "DIRECTORY"
```

Comment:
Correct, this change was made a long time ago, though. I missed it during my copy paste (if you look in the old API version it is already in all caps, and that change was announced and versioned and everything).